lib/lingui-config.js

```diff
@@ -10,12 +9,11 @@ import PathResolver from "../lib/path.js";
  */
 export async function getLinguiConfig({ config, cwd }) {
   try {
-    const resolver = new PathResolver({ cwd });
-    const resolvedContentFile = await resolver.resolve("./content");
```

Comment:
it seems like this just wasn't being used?

Reply:
> it seems like this just wasn't being used?

It was used, but decided it was better to just default to always use the same namespace for the compiled message files rather than trying to be smart about it.